### ISSUE TITLE
Randdomize CVs (also with no Repeats) are broken 

### ISSUE THREAD

**Issue by fractalgee** [patheros/JPManuals#6]
In Star Matrix  'Randomize CVs' right-click options (both) does nothing. This is JPLabs 2.0.2 in Rack 2.4.1 Pro on Mac arm64. Randomize Rhythms (and transitions )in Star Matrix do work however

**Comment by patheros** [comment]
@fractalgee currently Star Matrix Randomize CVs won't work unless you first set a note pool in the right click menu.

**Comment by fractalgee** [comment]
yep, I did figure that out after I opened this hence closed it again.....
